ext/standard/math.c

```diff
@@ -875,11 +877,39 @@ PHPAPI int _php_math_basetozval(zval *arg, int base, zval *ret)
 			c -= 'A' - 10;
 		else if (c >= 'a' && c <= 'z')
 			c -= 'a' - 10;
-		else
+		else if (isspace(c)) {
+			if (num == 0 && fnum == 0) {
+				continue;
```

Comment:
I think this is going to allow numbers like `"0 0"`, which is presumably not intended. Instead we should first skip leading whitespace before them main loop, and then treat them as any other invalid characters in here.